<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMain</class>
 <widget class="QWidget" name="FormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>81</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDOSet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set DO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDO">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClean">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDOSet">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>230</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAOSet">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>275</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1, 3.5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAO">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZeroing">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>85</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeroing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAOSet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set AO</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtOut">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
